### ISSUE TITLE
How do I export DB from V2 to V3 ?

### ISSUE THREAD

**Issue by Hydeish** [snipe/snipe-it#2784]
I'm trying to export the DB from my V2.1 to my newly installed V3 (v3.4.0-9-g67315d8).

I've tried both going via Admin/backup and command prompt, I've even tried doing a manualt export/import directly in MySQL Workbench.

What is the best way to copy all history, settings, assets etc to my new installation?

-----
**When I try via Admin/Backup I get** 
![image](https://cloud.githubusercontent.com/assets/22936840/19521486/23c7cd82-9614-11e6-8c2b-00c18b1c14d9.png)

**When I try php artisan db:backup on the cli**

![image](https://cloud.githubusercontent.com/assets/22936840/19521453/0410220a-9614-11e6-8863-5008b70b4a70.png)

When I import the backup I've done of the old site via mysql workbench the whole page renders error messages.

I've tried php artisan:migrate several times.

The path to the mysqldump is in my PATH system variables and in my .env file (not that it mattes for the old site).
-----

#### Please confirm you have done the following before posting your bug report:

- [ X] I have enabled debug mode
- [X ] I have read [checked the Common Issues page](http://docs.snipeitapp.com/common-issues.html)

-----
#### Please provide answers to these questions before posting your bug report:

- Version of Snipe-IT you're running **V2.1**
- What OS and web server you're running Snipe-IT on **Win 2008 R2 64-bit,** 
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc) **Manual**
- If you're getting an error in your browser, include that error **InvalidArgumentException**
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error **/snipe/admin/settings/backups**
- If a stacktrace is provided in the error, include that too.
 ```
 foreach ((array) $dirs as $dir) {
            if (is_dir($dir)) {
                $resolvedDirs[] = $dir;
            } elseif ($glob = glob($dir, GLOB_BRACE | GLOB_ONLYDIR)) {
                $resolvedDirs = array_merge($resolvedDirs, $glob);
            } else {
                throw new \InvalidArgumentException(sprintf('The "%s" directory does not exist.', $dir));
            }
        }
```
- Any errors that appear in your browser's error console.
- Confirm whether the error is [reproduceable on the demo](https://snipeitapp.com/demo). NO
- Include any additional information you can find in `app/storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

**Comment by Hydeish** [comment]
While I haven't been able to automatically import all my data I tried doing it manually. Did encounter some problems then.

I might have run composer as admin on the server.

I have manually extracted all relevant data (except history) from my sql server. 
I have added it to a CSV-file as per your template.

I get this error in the WebGUI 
![image](https://cloud.githubusercontent.com/assets/22936840/19557786/a72a4002-96c7-11e6-9c98-73e9bd85f430.png)

And this error using the cmd.

![image](https://cloud.githubusercontent.com/assets/22936840/19557853/e02c4292-96c7-11e6-8ff1-da2f89bab6be.png)
